### DISCUSSION TITLE
Geforce NOW

### DISCUSSION THREAD

**Discussion by 109302013evantsao** [1101-GS4719-C/Python-Community-GS4719-C#236] [General]
Geforce NOW是由NVIDIA推出的雲端串遊遊戲服務，他提供了另一種玩遊戲的選擇，
我們在用電腦時或玩遊戲時蠻常遇到硬體條件不夠的問題，換張顯示卡、CPU又要花好幾萬
使用Geforce NOW只需要有良好的網路速率，藉由網路傳輸運算後的結果，
不需要高要求的硬體配置，就可以享受到良好、高畫質、穩定的畫面體驗，
我認為這是一種未來的趨勢，不但可以減輕電腦負擔，也可以得到更好體驗
(這真的不是廣告，我只是推廣)
[geforce now](https://gfn.taiwanmobile.com/portal/index)

**Comment by 109802503gloria** [comment]
現在遊戲的質量越來越好，有這種服務讓玩遊戲的門檻又降低了~

**Comment by Isongzhe** [comment]
以目前Geforce NOW來說，除非你家網路能穩定100M以上，不然體驗不是很好。就算網速能穩定100M，舉例 APEX 還是會有延遲跟閃退。
